src/views/CreateEditHolder.js

```diff
@@ -132,7 +164,28 @@ export class EditArticle extends Component {
     }
   };
 
+  clearErrorMessage = () =>
+    setTimeout(() => this.setState({ error: "" }), 3000);
+
+  displayErrorMessage = error => (
+    <div className="error error__message">
+      {error.length ? <Error title={error} /> : false}
+    </div>
```

Comment:
use an empty string for displaying an error, and remove length checking. Get the error from props.